src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -49,6 +49,11 @@ enums:
         meaning: rdfs:Datatype
       rdf property:
         meaning: rdf:Property
+      sssom literal:
```

Comment:
from @cmungall: Using spaces in enums might lead to funny encoded spaces in RDF serialisations.

Reply:
All values on the `EntityType` enum (not only `sssom literal`) have had spaces since the beginning... Mildly opposed to changing that now.

Reply:
Agreed.